scripts/js/commands/checkInternalLinksFromOtherRepo.ts

```diff
@@ -19,6 +19,7 @@
  */
 
 import { readFile } from "fs/promises";
+import { readJsonFile } from "../lib/fs";
```

Comment:
Nit: we group our imports between standard library, third-party, and first-party. It helps to keep things organized. This should move to the section on lines 28-29 with the other first-party imports

Ditto the rest of the imports